app/models/concerns/crosscitable.rb

```diff
@@ -74,14 +74,14 @@ def datacite
     end
 
     def xml
-      datacite
+      current_metadata.xml
```

Comment:
This changes breaks the conversion of other input formats to DataCite XML.

Reply:
I'm unclear on this one. If the default storage format is DataCite XML, why do we need a conversion step in the getter of the XML field ??

Reply:
You are right. I was thinking that in some cases we might do a conversion of the DataCite XML, but not here.